examples/03_evaluate/README.md

```diff
@@ -20,6 +21,11 @@ Two approaches for evaluating model performance are demonstrated along with thei
 3. Classification metrics: These are used to evaluate binary labels
     * Arear Under Curver (AUC) - integral area under the receiver operating characteristic curve
     * Logistic loss (Logloss) - the negative log-likelihood of the true labels given the predictions of a classifier
+4. Non accuracy based metrics: These do not compare predictions against ground truth but instead evaluate the following properties of the recommendations
+    * Novelty - measures of how different an item is compared to the items that users have already known or interacted with 
```

Comment:
* Novelty - measures of how novel recommendation items are by calculating their recommendation frequency among users who have not interacted with these items  
 * Diversity - measures of how different items in a recommendation set are with respect to each other
 * Serendipity - measures of how surprising recommendations are to a specific user by comparing them to the items that the user have already interacted with 
 * Coverage - measures related to the distribution of items recommended by the system

Reply:
Novelty is calculated for each item respect to all users; Serendipity is calculated for each item respect to a specific user. These two concepts tend to get confused.

Reply:
Good suggestions, I changed these.